hack/test-cmd.sh

```diff
@@ -46,7 +46,7 @@ set -e
 
 function find_tests {
   cd "${OS_ROOT}"
-  find "${1}" -name '*.sh' -print0 | sort -u | xargs -0 -n1 printf "%s\n"
+  find "${1}" -name '*.sh' -print0 | sort -uz | tr '\0' '\n'
```

Comment:
we already use the script name as a project name, so it can't contain spaces, etc